src/utils/plot_results.jl

```diff
@@ -54,7 +54,10 @@ function get_stacked_plot_data(res::OperationsProblemResults, variable::String;
 
     data_matrix = convert(Matrix, variable)
     labels = collect(names(variable))
-    legend = string.(labels)
+    legend = [names(variable)[1]]
```

Comment:
`names` is called twice. You could use `labels` in the second case.  Also, should you verify that the length of `labels` is greater than 2?  Or is that guaranteed?